packages/block-editor/src/components/toolbar-block-appender/index.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * WordPress dependencies
+ */
+import { ToolbarButton, ToolbarGroup } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import __unstableBlockToolbarLastItem from '../block-toolbar/block-toolbar-last-item';
+import { useBlockEditContext } from '../block-edit/context';
+import Inserter from '../inserter';
+
+function ToolbarBlockAppender( {
+	defaultBlock = { name: 'core/paragraph', attributes: {} },
```

Comment:
Instead of hardcoding paragraph here, should pull dynamically current default block for the current block.